src/binder/statement/select_statement.cpp

```diff
@@ -1,37 +1,325 @@
 #include "binder/statement/select_statement.h"
```

Comment:
This is the last file I am reviewing and I am starting to become sleepy again, so it'll be very lightly reviewed..

Reply:
Indeed this is one of the most significant change of this PR 🤣 I'll apply best practice to this file.